src/showcase/assets/stackblitz/src/app/app.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { SbbModule } from './sbb.module';
+import { {0} } from './{1}';
+
+@NgModule({
+  imports:      [ BrowserModule, BrowserAnimationsModule, FormsModule, ReactiveFormsModule, SbbModule ],
```

Comment:
Add the SbbIconModule from '@sbb-esta/angular-core/icon' and also the provider SBB_ICON_REGISTRY_PROVIDER (See the showcase app.module.ts for comparison)